style.css

```diff
@@ -118,6 +130,9 @@ body {
     position: absolute;
     height: 100%;
     background: transparent;
+    color: var(--secondary-white);
```

Comment:
Will suggest to rethink color variables like these. Are you using some naming convention?

Reply:
Not any as of now. WIll refactor down the line